src/components/body/Body.jsx

```diff
@@ -0,0 +1,13 @@
+import Navbar from "src/components/navbar/Navbar";
+import Content from "src/components/content/Content";
+
+const Body = () => {
+  return (
+    <div>
+      <Navbar />
+      <Content />
+    </div>
```

Comment:
you can use <></> if div isn't necessary
```suggestion
    <>
      <Navbar />
      <Content />
    </>
```

Reply:
Please try to write `<Fragment>` instead of `<>`